EnglishFilter/sections/foreign.txt

```diff
@@ -851,6 +851,8 @@ javfree.club#%#(function() { var w_open = window.open, regex = /agxclick\.com/;
 ! https://github.com/AdguardTeam/AdguardFilters/issues/33453
 85tube.com##.block-video > .table
 ||85tube.com/player/html.php?aid=*_html&video_id=*&*&referer
+! https://github.com/AdguardTeam/AdguardFilters/issues/31528
+r3sub.com##.container a[target="_blank"] > img
```

Comment:
Can you specify the rule a bit more please. Sadly the site is not opening for me.

Reply:
Try Hong Kong VPN to open this site. Now this rule is redundant cause i can't reproduce ad block.

Reply:
That means we can close this pull request?

Reply:
Yes, now it's clean for me.

Reply:
Yesterday i can't reproduce it, but today i see just a one banner.